test/aave-stop-loss-multiply.ts

```diff
@@ -43,9 +44,16 @@ describe('AaveStoplLossCommand-Multiply', async () => {
             ],
         })
         const system = await deploySystem({ utils: hardhatUtils, addCommands: true })
-        const guardDeployerAddress = '0x060c23f67febb04f4b5d5c205633a04005985a94'
+        account = (await hre.ethers.getContractAt(
+            'IAccountImplementation',
+            testPositionAddress,
+        )) as IAccountImplementation
+        const oldGuard = await account.guard()
+        await hardhatUtils.replaceImmutableAddress(hardhatUtils.addresses.DPM_GUARD, oldGuard, system.dpmAdapter!)
```

Comment:
this is about to brake when hardhatUtils.addresses.DPM_GUARD is changed since blocknumber is fixed

Reply:
It should not. We take the old address of dpm guard - using `account.guard()` and then replace it with current one.